autrainer/datasets/aibo.py

```diff
@@ -110,16 +117,50 @@ def __init__(
                 self.df_train, self.train_transform
```

Comment:
@ramppdev I am wondering if this is needed..

Since `self.train_dataset`, `self.dev_dataset`, and `self.test_dataset` are now `@cached_property`, they will be instantiated only once, when the user first calls them, which will happen after the `__init__` of the dataset which computes the `Standardizer` and adds it to the transforms..So it should be fine to remove this line now?

Reply:
Yes still needed due to `train_data = torch.cat([x for x, *_ in self.train_dataset])` instantiating the dataset, e.g. after that we need to manually pass the updated transform again. For dev and test its fine since these are not called yet.